src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -3589,6 +3637,122 @@ NodeIsRangeTblRefReferenceTable(Node *node, List *rangeTableList)
 }
 
 
+/*
+ * FetchEqualityAttrNumsForRTE fetches the attribute numbers from quals
+ * which has equality operator
+ */
+List *
+FetchEqualityAttrNumsForRTE(Node *node)
+{
+	if (node == NULL)
+	{
+		return NIL;
+	}
+	if (IsA(node, List))
+	{
+		return FetchEqualityAttrNumsForList((List *) node);
+	}
+	else if (IsA(node, OpExpr))
+	{
+		return FetchEqualityAttrNumsForRTEOpExpr((OpExpr *) node);
+	}
+	else if (IsA(node, BoolExpr))
+	{
+		return FetchEqualityAttrNumsForRTEBoolExpr((BoolExpr *) node);
+	}
+	return NIL;
+}
+
+
+/*
+ * FetchEqualityAttrNumsForList fetches the constant equality numbers
+ * from the given node list.
```

Comment:
constant equality numbers -> attribute numbers of expression of the form <attribute> = constant from the given node list.